runtimes/spiritnet/src/xcm_config.rs

```diff
@@ -252,19 +258,40 @@ impl cumulus_pallet_xcm::Config for Runtime {
 
 impl cumulus_pallet_xcmp_queue::Config for Runtime {
 	type RuntimeEvent = RuntimeEvent;
-	type XcmExecutor = XcmExecutor<XcmConfig>;
 	type ChannelInfo = ParachainSystem;
 	type VersionWrapper = PolkadotXcm;
-	type ExecuteOverweightOrigin = EnsureRoot<AccountId>;
 	type ControllerOrigin = EnsureRoot<AccountId>;
 	type ControllerOriginConverter = XcmOriginToTransactDispatchOrigin;
 	type WeightInfo = cumulus_pallet_xcmp_queue::weights::SubstrateWeight<Self>;
 	// TODO: Most chains use `NoPriceForMessageDelivery`, merged in https://github.com/paritytech/polkadot-sdk/pull/1234.
-	type PriceForSiblingDelivery = ();
+	type PriceForSiblingDelivery = NoPriceForMessageDelivery<ParaId>;
+	type MaxInboundSuspended = ();
```

Comment:
Maybe the comment was misleading. I thought this was what we wanted here since most chains are using it. I also believed the `NoPriceMessageDelivery` struct maintained the current logic, and the `ConstantPrice` would be a new feature.

Reply:
No I meant for the `MaxInboundSuspended` type.

Reply:
… Ups, seems like I forgot to set a meaningful number. Good catch. [f1a7a82](https://github.com/KILTprotocol/kilt-node/pull/655/commits/f1a7a82171d888503f05e223fcb0fec3260fefda) :grimacing: